packages/path_provider/path_provider_foundation/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 2.5.0
```

Comment:
Should this be a breaking change? Developers who are importing the "path_provider_foundation" module directly will get a build failure since it's not longer available. Here's an example I found:

https://github.com/MixinNetwork/flutter-app/blob/1a0fd1691c3c5fd9a4d86097d73efeb438cd9ca5/macos/Runner/AppDelegate.swift#L5